sklearn/preprocessing/imputation.py

```diff
@@ -377,3 +386,8 @@ def transform(self, X):
             X[coordinates] = values
 
         return X
+
+    @staticmethod
+    def _isnan(x):
+        """Check whether x is nan or not."""
+        return np.isnan(x) if isinstance(x, type(np.nan)) else False
```

Comment:
Why didn't np.isnan just work?

Reply:
Because by using `np.isnan` I got tests which failed. I guess there were some side effects. By using the statistics `np.isnan` works well.